angular/src/components/ciphers.component.ts

```diff
@@ -42,6 +42,10 @@ export class CiphersComponent {
         await this.load(filter, deleted);
     }
 
+    async refresh() {
```

Comment:
should this have the same `try/finally` that was removed?

Reply:
yes, good point

Reply:
I don't think so, the `try/finally` was only to set `this.refreshing` in the `finally` block. `this.refreshing` is only used for paging and is not needed for virtual-scroll. (`web` has its own `refresh` method with the original logic, because it still uses paging, but other clients don't need it.)